datus/agent/agent.py

```
@@ -889,6 +892,78 @@ def run_single_spider2_task(task_config):
         )
 
+        # new benchmark for test
+    def benchmark_tencent(self, benchmark_path, target_task_ids):
+       
+        json_file_path = os.path.join(benchmark_path, "after.json") 
+        logger.info(f"Loading custom benchmark file from: {json_file_path}")
+
+        try:
+            with open(json_file_path, 'r', encoding='utf-8') as f:
+                processed_tasks = json.load(f) 
+        except Exception as e:
```

Ruff (0.14.2)

Do not catch blind exception: `Exception`

(BLE001)